src/utils/validation.ts

```diff
@@ -76,3 +102,18 @@ export const validateBlockDirectory = (): void => {
         process.exit(1);
     }
 };
+
+export const validateIntegration = (
```

Comment:
I think we need to change the name of this one or have a separate function for translating the name to id. What do you think?

Reply:
I think splitting out the functions would be tricky, so I'll try a name of `integrationIdFromName`